### PR TITLE
Query window enhancement

### LINKED ISSUE #1550: Result window issues
- [ ] Check size on 1.6
- [ ] Add margin bellow results including scrollbar
- [ ] Set fixed height
- [ ] Don't display empty fields

### PR DESCRIPTION
**Pull request by olivierSemet**
Close #1550 #1549

**Demo**: 

- https://oliviersemet.github.io/ngeo/queryWindow-enhancement/examples/contribs/gmf/apps/desktop/

- https://oliviersemet.github.io/ngeo/queryWindow-enhancement/examples/contribs/gmf/apps/mobile/

### CODE REVIEW

**Comment by sbrunner** [comment]
On my results I have 5 attributes, and a scrollbar for about 1 px :-(

Also that's looks good :-)

**Comment by olivierSemet** [comment]
@sbrunner  you mean the scroll bar is displayed even if the overflow is ridiculously short?

**Comment by sbrunner** [comment]
Yes :-)

**Comment by ger-benjamin** [comment]
Note:  Keep your code like this. But for the displayQueryGrid, I (must) work on a system that excludes empty properties (Point "Don't display empty fields" in this related issue).

**Comment by olivierSemet** [comment]
@sbrunner now, the qWindow will display up to 8 properties on mobile and desktop without the need to scroll. On landscape orientation (mobile), only 4.

**Comment by sbrunner** [comment]
For me that's looks good, @pgiraud merge if it's OK for you :-)

**Comment by olivierSemet** [comment]
@pgiraud could we merge?

File: contribs/gmf/less/displayquerywindow.less [hunk 1]
```diff
@@ -211,3 +214,15 @@
     }
   }
 }
+
+@media (min-width : 321px) {
```
Comment (pgiraud) [inline contribs/gmf/less/displayquerywindow.less]: Where does this value "321px" come from?

**Comment by olivierSemet** [comment]
@pgiraud check this better solution with the use of orientation media parameter

File: contribs/gmf/less/displayquerywindow.less [hunk 2]
```diff
@@ -1,6 +1,9 @@
 @import "../../../node_modules/font-awesome/less/variables.less";
 
 @displayquerywindow-tablet-width: 8 * @map-tools-size;
+@displayquerywindow-detailed-header-height : 5rem;
```
Comment (pgiraud) [inline contribs/gmf/less/displayquerywindow.less]: no space before colon

**Comment by pgiraud** [comment]
One last minor comment. Otherwize looks good.

**Comment by olivierSemet** [comment]
Thank you for review @pgiraud. Following your comments I pushed a new version

**Pull request merged**